chassis/core/src/test/resources/spring/example.test.suite.conf.xml

```diff
@@ -4,7 +4,7 @@
        xmlns:context="http://www.springframework.org/schema/context"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd">
 
-    <context:component-scan base-package="com.griddynamics.jagger"/>
+    <context:component-scan base-package="com.griddynamics.jagger" resource-pattern="ExampleJLoadScenarioProvider.class"/>
```

Comment:
This class can be named not only ExampleJLoadScenarioProvider, why it's needed here?